deployment/provision_vagrant_vm.sh

```diff
@@ -100,7 +109,7 @@ apt-get -q install -y libasound2 libgconf-2-4 libgbm1 libgtk-3-0 libnss3 libx11-
 wget https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh --no-verbose --output-document - | sudo -H -u $USER bash
 
 # setup evap
-cd "$MOUNTPOINT"
+cd "$REPO_FOLDER" || exit 1
```

Comment:
Why the change to `REPO_FOLDER` here? Is it important?

Reply:
`MOUNTPOINT` may have incorrect ownership, $REPO_FOLDER is guaranteed to be correct because of bindfs

Reply:
Specially, all calls with `sudo -H -u $USER ...` will fail then